projects/plugins/jetpack/modules/masterbar/admin-menu/class-admin-menu.php

```diff
@@ -396,6 +396,34 @@ public function add_options_menu() {
 
 		$this->update_submenus( 'options-general.php', $submenus_to_update );
 
+		if (
+			/**
+			 * Filter to enable the Newsletter Settings section in Calypso UI.
+			 *
+			 * @since $$next-version$$
+			 * @module masterbar
+			 *
+			 * @param bool false Enable newsletter setting section? Default to false.
+			 */
+			apply_filters( 'calypso_use_newsletter_settings', '__return_false' )
```

Comment:
@simison , I think you may have mixed up the second parameter of `apply_filters` with the second argument of the `add_filter` call. The second argument of `apply_filters` is meant to be a value that is passed to the filter for processing, in case where the filter does not exist the value is returned back. In this case the '__return_false' is returned which passes the condition:
```php
apply_filters( 'calypso_use_newsletter_settings', false)
```
should fix the issue.